bloom-housing/bloom
# 3211/five bedroom listings

# Pull Request Template

## Issue Overview

This PR addresses #3211 


## Description

PR adds option for 5 Bedroom units, and expands occupancy to 11 (from 10). I updated also some inconsistent translations for other unit types.

## How Can This Be Tested/Reviewed?

Run migration or reseed db. Go to listing (in partners) with open application process. Add another unit (or edit exiting) in select field `unit type` there should be now another value for `Five bedroom` also select items will be now sorted using `numBedrooms`. In same place `Max occupancy` should have additional option `11`. To further testing will be good to have at least `3 BR`, `4 BR`, and `5 BR`.
In public we should see those units in listings view and after clicking certain listing.
Last check will be in application process in Household step (those translation might be different than one in tables for example `5BR` in table , and `5 Bedroom` in application.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
